test/io/lvh/silly_solve_expresso_oracle_test.clj

```diff
@@ -0,0 +1,366 @@
+(ns io.lvh.silly-solve-expresso-oracle-test
+  (:require
+   [clojure.test :as t]
+   [clojure.test.check.generators :as gen]
+   [clojure.test.check.properties :as prop]
+   [clojure.test.check.clojure-test :refer [defspec]]
+   [io.lvh.silly-solve :as ss]
+   [numeric.expresso.core :as ex]
+   [clojure.walk :as walk]))
+
+;; =============================================================================
+;; Expresso Integration Utilities
+;; =============================================================================
+
+(defn keyword->symbol
+  "Convert a keyword to a symbol. Expresso works better with symbols for multi-variable solving."
+  [k]
+  (if (keyword? k)
+    (symbol (name k))
+    k))
+
+(defn symbol->keyword
+  "Convert a symbol back to a keyword for silly-solve compatibility."
+  [s]
+  (if (symbol? s)
+    (keyword (name s))
+    s))
+
+(defn convert-expr-keywords->symbols
+  "Recursively convert all keywords in an expression to symbols."
+  [expr]
+  (walk/postwalk
+   (fn [x]
+     (if (keyword? x)
+       (keyword->symbol x)
+       x))
+   expr))
+
+(defn convert-result-symbols->keywords
+  "Convert symbol keys in a result map back to keywords."
+  [result-map]
+  (when (map? result-map)
+    (into {} (map (fn [[k v]] [(symbol->keyword k) v]) result-map))))
+
+(defn extract-variables-from-equations
+  "Extract all variables from a list of equations."
+  [equations]
+  (let [operators #{'+ '- '* '/ '** 'max 'min '=}]
```

Comment:
Instead of having a hardcoded list of operators ostensibly we could extract this from upstream's `ops`, on the other hand I guess that would make the test more brittle because we've already talked about modifying that system so it's user-exstensible...